src/test/java/org/checkerframework/specimin/SpeciminTestExecutor.java

```diff
@@ -69,9 +70,14 @@ public static void runTest(
       speciminArgs.add("--targetFile");
       speciminArgs.add(targetFile);
     }
-    for (String targetMethod : targetMethods) {
-      speciminArgs.add("--targetMethod");
-      speciminArgs.add(targetMethod);
+    for (String targetMember : targetMembers) {
```

Comment:
This is the most elegant approach I can come up with that does not require modifying all the existing 148 test files.

Reply:
This seems fine to me for now. We can come back to this after the deadline if you don't think it's elegant enough.